packages/labs/analyzer/src/lib/lit-element/lit-element.ts

```diff
@@ -113,12 +113,11 @@ export const isLitElementSubclass = (
   const checker = analyzer.program.getTypeChecker();
   const type = checker.getTypeAtLocation(node) as ts.InterfaceType;
   const baseTypes = checker.getBaseTypes(type);
-  for (const t of baseTypes) {
-    if (_isLitElementClassDeclaration(t, analyzer)) {
-      return true;
-    }
-  }
-  return false;
+  return baseTypes.some((t) =>
+    t.isIntersection()
```

Comment:
Let's leave a comment for why this is here:
```suggestion
    // Mixins will cause the base types to be an intersection that
    // includes `LitElement`
    t.isIntersection()
```